fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -187,6 +194,20 @@ private void checkRebuildProps() {
         }
     }
 
+    /**
+     * Check for deprecated properties and log warnings if they are used
+     */
+    private void checkDeprecatedProperties() {
+        if (System.getProperty(FCREPO_REBUILD_CONTINUE) != null) {
+            LOGGER.warn("The property '{}' is deprecated and will be removed in a future version. " +
+                    "Use {} instead.", FCREPO_REBUILD_CONTINUE, FCREPO_REBUILD);
+        }
+        if (System.getProperty(FCREPO_REBUILD_ON_START) != null) {
+            LOGGER.warn("The property '{}' is deprecated and will be removed in a future version. " +
+                    "It now behaves the same as {}.", FCREPO_REBUILD_ON_START, FCREPO_REBUILD);
+        }
+    }
```

Comment:
Are you going to set the `rebuildEnabled` property based on the two deprecated properties, if not I would adjust the second message above to have the same "Use {} instead". It sounds more...required.